### PR TITLE
move setText method code to write method

### PR DESCRIPTION
**Pull request by theoweiss**
Fix for #5091

### CODE REVIEW

**Comment by theoweiss** [comment]
@JamesDodds tested a snapshot and it worked.
From my point of view the PR is ready to merge.

File: bundles/binding/org.openhab.binding.tinkerforge/src/main/java/org/openhab/binding/tinkerforge/internal/model/impl/MBrickletLCD20x4Impl.java [hunk 1]
```diff
@@ -1029,10 +971,70 @@ public void initSubDevices() {
...
     @Override
-    public void write(String text) {
-        // TODO: implement this method
-        // Ensure that you remove @generated or mark it @generated NOT
-        throw new UnsupportedOperationException();
+    public void write(String newText) {
+        String oldText = text;
+        text = newText;
+        // TODO check if there are changes
+        logger.debug("setText: " + newText);
```
Comment (9037568) [inline bundles/binding/org.openhab.binding.tinkerforge/src/main/java/org/openhab/binding/tinkerforge/internal/model/impl/MBrickletLCD20x4Impl.java]: Could you convert this to a parameterized call?

**Comment by 9037568** [comment]
Aside from the one comment, this LGTM.

**Comment by 9037568** [comment]
Thanks, @theoweiss !

**Comment by watou** [comment]
(@9037568 please cherry pick after merging onto master those PRs marked for maintenance releases.  I happened to notice this one so I will cherry-pick it myself.  Thanks!)

**Comment by watou** [comment]
Cherry picked onto 1.9.x

**Comment by 9037568** [comment]
@watou: Thanks. I will watch out for that in future.

**Pull request merged**